GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/BasicFileSystemTests.cs

```diff
@@ -941,7 +935,38 @@ public void RunPythonExecutable()
             result.Output.ShouldContain("3.14");
 
             GitProcess.Invoke(this.Enlistment.RepoRoot, "checkout " + this.Enlistment.Commitish);
-       }
+        }
+
+        private void CheckForDeletedFile(string filePath, FileSystemRunner fileSystem)
+        {
+            if (this.SupportsPosixDelete())
+            {
+                filePath.ShouldNotExistOnDisk(fileSystem);
+            }
+            else
+            {
+                filePath.ShouldBeAFile(fileSystem);
+            }
+        }
+
+        private bool SupportsPosixDelete()
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // https://msdn.microsoft.com/en-us/library/windows/desktop/ms724429(v=vs.85).aspx
+                FileVersionInfo kernel32Info = FileVersionInfo.GetVersionInfo(Path.Combine(Environment.SystemDirectory, "kernel32.dll"));
+
+                // 18362 is first build with posix delete support in windows

```

Comment:
Tiny nit: This is the first release where the DeleteFile API uses POSIX style deletes by default (earlier versions supported POSIX delete but it was not the default).